uaclient/cli.py

```diff
@@ -227,17 +230,13 @@ def action_attach(args, cfg):
     else:
         contract_token = args.token
 
-    machine_token_response = contract_client.request_contract_machine_attach(
-        contract_token=contract_token)
-    contractInfo = machine_token_response['machineTokenInfo']['contractInfo']
-    for entitlement in contractInfo['resourceEntitlements']:
-        if entitlement.get('entitled'):
-            # Obtain each entitlement's accessContext for this machine
-            entitlement_name = entitlement['type']
-            contract_client.request_resource_machine_access(
-                cfg.machine_token['machineToken'], entitlement_name)
+    if not contract.request_contract_updates(cfg, contract_token):
+        print(
+            "Could not attach machine. Error contacting server %s" %
```

Comment:
Should this string be in `ua_status` instead?

Reply:
tried to move some strings to ua_status

Reply:
moved a few more strings to ua_status.MESSAGE_*